source/code-snippets/aggregation/agg.js

```diff
@@ -0,0 +1,48 @@
+const { MongoClient, ExplainVerbosity } = require('mongodb');
+const uri = 'mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority';
+const client = new MongoClient(uri);
+
+async function run() {
+    try {
+        await client.connect();
+
+        // begin data insertion
+        const db = client.db("aggregation");
+        const coll = db.collection("restaurants");
+
+        const docs = [
+            { "name": "Rising Sun Bakery", "stars": 3, "categories": ["Bakery", "Cafe", "Italian"] },
+            { "name": "Cafe au Late", "stars": 4, "categories": ["Bakery", "Cafe", "Coffee", "Dessert"] },
+            { "name": "Liz's Coffee Bar", "stars": 5, "categories": ["Coffee", "Cafe", "Bakery"] },
+            { "name": "Oak Steakhouse", "stars": 3, "categories": ["Steak", "Seafood"] },
+            { "name": "Petit Cookie", "stars": 4, "categories": ["Bakery", "Cookies", "Cake", "Coffee"] },
```

Comment:
Could the "name" fields be moved to the end of the documents? I think this will make the documents easier to scan, as the "stars" and "categories fields will be aligned:

```suggestion
            { "stars": 3, "categories": ["Bakery", "Cafe", "Italian"],  "name": "Rising Sun Bakery" },
            { "stars": 4, "categories": ["Bakery", "Cafe", "Coffee", "Dessert"], "name": "Cafe au Late" },
            { "stars": 5, "categories": ["Coffee", "Cafe", "Bakery"], "name": "Liz's Coffee Bar"},
            { "stars": 3, "categories": ["Steak", "Seafood"], "name": "Oak Steakhouse" },
            { "stars": 4, "categories": ["Bakery", "Cookies", "Cake", "Coffee"], "name": "Petit Cookie"},
```

Reply:
I think that presents the data in a weird way, as we're usually accustomed to the identifying details being first (like names or id fields)